### ISSUE TITLE
load further images after creating project/task ?

### ISSUE THREAD

**Issue by Petros626** [opencv/cvat#4658]
Hello,

I am looking for the option to expand the record after I have already uploaded images.  Is it only possible when creating a new task or can I upload images afterwards?

**Comment by nmanovic** [comment]
@Petros626 , you always can create more tasks in a project. It is a way to add images to your dataset (project == dataset).

**Comment by Petros626** [comment]
Hello Nikita,

thank you for the quick answer, so it's only possible through an new task not an existing.

**Comment by nmanovic** [comment]
@Petros626 , yes, it is. It is dangerous to change an existing task because it will invalidate data annotations and so on. Thus after you create a task, it is not possible to change the number of images (in the future it will be possible to mark a frame as deleted). Use a project as a dataset with a number of tasks. Each task is a portion of data which you want to annotate. Each task can be spliced on multiple jobs to annotate it in parallel by multiple annotators.

**Comment by GrigoriiTarasov** [comment]
"Each task can be spliced on multiple jobs to annotate it in parallel by multiple annotators."

Yes, but is it a way to create task by API v1? (can't check if images was really loaded without that)

1) creating new with
`requests.post(...) `on `/api/v1/tasks` with 
segments 
```
[{'start_frame': 0,
  'stop_frame': 62,
  'jobs': [{'assignee': None, 'reviewer': None, 'status': 'annotation'}]}]
```
shows 'No data' in jobs section
2) `requests.patch(...) ` on `'/api/v1/tasks/{}'.format(new_task_id)`
with same segments has same result: no jobs actually created.

**Comment by nmanovic** [comment]
@Petros626 , I hope we answered on your question. I will close the issue.
